docs/DeveloperGuide.md

```diff
@@ -1,34 +1,851 @@
+function will return true.
+
+Step.7 Within the `minusTime()` function, it will then check if the hours the user wish to remove is more than the
+existing hours by
+calling the `doesHoursExceedTotal()` function. If the module has an existing workload, the `doesHoursExceedTotal()`
+function will return true.
+
+Step.8 A time of `2` hours in academic week `4` will now be removed from the actual workload of the `CS2113T` module. 
+
+Step.9 The system will print the string `2 hours have been removed from CS2113T`
+
+Step.10 The actual workload is updated in storage.
+
+The following sequence diagram illustrates what happens when a user executes `minusTime CS2113T 2 4`.
+
+[insert diagram here]
+
+
+#### Design Considerations
+
+* **Alternative 1 (current choice)**: removing time with academic weeks associated
+    * Pros: Easy to manage for user as the user only needs to focus on the current academic week to track his workload
+    * Cons: User might prefer to manage his time spent in totality
+* **Alternative 2**: minus time in totality 
+    * Pros: Able to cumulatively remove time and manage workload based on overall total time spent.
+    * Cons: May not be useful to know total time spent as workload may be better managed weekly.
+
+### 4.5 View modules <a name="4.5"></a>
+
+#### Current Implementation
+
+The view module command allows the user to view all the modules taken,
+the respective time spent and expected module workload at a glance. It prints 
+`weekNumber`,`moduleCode`, `expectedWorkload` and `actualWorkload` in the specified 
+week for all the modules taken in a table format. 
+
+Given below is an example usage scenario and how the view module command behaves at each step.
+
+1. User calls the view module command from the `Parser`. `Parser` then calls the `printTable()`
+method in `Ui`. `printTable()` then instantiates `ModView` class and calls the `printAllModuleInformation()`
+method from the `ModView` class.
+
+1. User input is subjected to `validateInputs()` to verify that 
+both of the following conditions are satisfied.
+  * `weekNumber` is an integer between 1 and 13 inclusive.
+  * `modlist` in `list` is not empty.
+
+If any of the above conditions is not satisfied, an error message will be printed
+and `printAllModuleInformation()` terminates.
+
+1. A method `getMaxModuleLength()` will be called to find out the maximum length 
+of module code of modules taken. This is to facilitate the resizing of the printed table.
+
+1. `updateTemplates()` then updates the table templates such as `border`, `header` 
+and `contents` are updated accordingly based on the output of `getMaxModuleLength()`.
+
+1. For each `Module` in `modList`, `updateContent()` will be called to update the contents
+to be printed.
+
+The following sequence diagram shows how the view module command works. 
+
+![view-module](diagrams/ModViewSequence.png)
```

Comment:
Shouldn't this be alt since its the official language?
![review4 6](https://user-images.githubusercontent.com/57140494/97388584-5bca7500-1913-11eb-83a3-1130250c35cf.png)